### DISCUSSION TITLE
最新的 迅雷 algorithms 参数

### DISCUSSION THREAD

**Discussion by sang8052** [AlistGo/alist#4654] [Show and tell]
感谢大佬们提供迅雷云盘的captcha_sign 的计算方法，这为我抓包提供了思路
所以在这里 公布一下最新的 迅雷的 captcha_sign 计算时使用的 algorithms  参数，
目前适配 迅雷云盘 web 版 client_version 的值为 1.82.0 的 客户端
为防止被搜索引擎匹配 ， algorithms  参数使用 base64 编码后上传 ， 使用时请 base64 解码 
[base64 在线解码](https://base64.us/)
[迅雷云盘专家版](https://alist.nn.ci/zh/guide/drivers/thunder.html#%E8%BF%85%E9%9B%B7%E4%B8%93%E5%AE%B6%E7%89%88)
6L+F6Zu35LqR55uY572R6aG154mIIDEuODIuMCBhbGdvcml0aG1zIOWPguaVsAphdXRob3I6c3VkZW1xYXEgZW1haWw6bWFpbEBzemhjbG91ZC5jbiAKYjlEbGR2NmtSc1J5T0c0dEZIemVKNFJiT2kwbjduTzhvbUZvdUxWZ3ZMTkIuVEVIRE90ZU1QclJCNjZ5UUlGOXRGK3BmUEFJZXNhL3hnLkZteDI3R2xOYnJJeGlQU1FWbS5jcmxQVnJpUFJBaXVDRUtadks0eWloUDU1Z1RSdkxkN3FEVkxzRHRXemhrWHQ1SXFzN1Rwb1AuRTJ0b29nc2VFZGdYbWxmbnoxcHBVaFV2RDlCMmpnU0ErWUcuYTJmM0wwQWlvVSswUHZUZUN0ay42ZDZ3MXhYOWo5NUdFUE5wZCtUNEhtYlRjZVpORUYzMTBwcFJlLkJ2c0orQ1NTN2kuUnYK

**Comment by ChangMM** [comment]
@sang8052 试了下 好像不行了
